src/store/plugins/names.js

```diff
@@ -155,6 +156,9 @@ export default (store) => {
             { root: true },
           );
         } catch (e) {
+          if (isInsufficientBalanceError(e)) {
+            e.message = i18n.t('modals.insufficient-balance.msg');
+          }
```

Comment:
I suggest to avoid modifying error message property. Instead I suggest to change line 162 in the following way:
```
dispatch('modals/open', {
  name: MODAL_DEFAULT,
  msg: (isInsufficientBalanceError(e))
    ? i18n.t('modals.insufficient-balance.msg')
    : e.message
}, { root: true });
```

Reply:
Thanks